eZ/Publish/API/Repository/Values/URL/Query/Criterion/VisibleOnly.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace eZ\Publish\API\Repository\Values\URL\Query\Criterion;
+
+/**
+ * Matches URLs which are visible.
```

Comment:
Expand on what you mean by visible URL's

Reply:
Done in c74b7acf267b60dea71b5b3798221957eb64ffbe